app/views/content/life-as-a-teacher/explore-subjects/drama/_article.html.erb

```diff
@@ -0,0 +1,73 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach drama</h2>
+    <div class="inset">
+      <p>In drama lessons, you’ll foster a safe space for pupils to explore ideas creatively and help them bring characters to life. They’ll build skills in teamwork and empathy, take bold risks, and discover their own distinct voices.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_progression_chris' %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+    <div class="inset">
+      <p>Teaching drama opens the door for your pupils to pursue exciting careers in fields such as acting, stage management and set design.</p>
+      <p>Your pupils will learn more from you than you might think. As their drama teacher, you'll guide them through the skills needed to write, rehearse and perform dramatic pieces. You'll teach them valuable life skills such as confidence with public speaking, working collaboratively and thinking creatively.</p>
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_drama_itasha' %>
+  </section>
+
+  <section class="col col-720">
+   <h2 class="heading--box-blue">What you'll teach</h2>
+   <p>Drama is part of the spoken language element of the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-english-programmes-of-study/national-curriculum-in-england-english-programmes-of-study#key-stage-3">national curriculum for English</a>.</p>
```

Comment:
@joegibb I'm not sure we should link to an anchor point way down the page - it's not best practice and users will miss all the introduction information on Spoken language at the top of the page? But I will leave this for you and @Sarah-DfE to decide